stockflux-launcher/src/App.js

```diff
@@ -1,6 +1,7 @@
 import React, {useRef, useEffect} from 'react';
 import cx from 'classnames';
 import 'stockflux-components';
+import {OpenfinApiHelpers} from 'stockflux-core';
```

Comment:
Should this be changed to `{ OpenfinApiHelpers }` since that is the convention that seems to be used everywhere else?